server/services/netatmo/lib/event/netatmo.newValueCamera.js

```diff
@@ -0,0 +1,245 @@
+      newCamera = {
+        name: data.name,
+        should_poll: false,
+        external_id: `netatmo:${sid}`,
+        selector: `netatmo:${sid}`,
+        service_id: this.serviceId,
+        model: `netatmo-${data.type}`,
+        cameraUrl: {
+          name: `CAMERA_URL - ${data.name}`,
+          value: `${data.vpn_url}/live/snapshot_720.jpg`,
+        },
+        features: [
+          {
+            name: data.name,
+            selector: `netatmo:${sid}:camera`,
+            external_id: `netatmo:${sid}:camera`,
+            category: DEVICE_FEATURE_CATEGORIES.CAMERA,
+            type: DEVICE_FEATURE_TYPES.CAMERA.IMAGE,
+            read_only: false,
+            keep_history: false,
+            has_feedback: false,
+            min: 0,
+            max: 0,
+          },
+          {
+            name: `Power - ${data.name}`,
+            selector: `netatmo:${sid}:power`,
+            external_id: `netatmo:${sid}:power`,
+            category: DEVICE_FEATURE_CATEGORIES.SWITCH,
+            type: DEVICE_FEATURE_TYPES.SWITCH.BINARY,
+            read_only: true,
+            keep_history: true,
+            has_feedback: true,
+            min: 0,
+            max: 1,
+          },
+        ],
+        params: [
+          {
+            name: `CAMERA_URL - ${data.name}`,
+            value: `${data.vpn_url}/live/snapshot_720.jpg`,
+          },
+        ],
+      };
+      if (data.modules !== undefined) {
+        data.modules.forEach((module) => {
+          if (module.type === 'NIS' || module.type === 'NACamDoorTag') {
+            // if the "NACamera" camera module is present
+            const sidModule = module.id;
+            const moduleName = module.name;
+            logger.debug(`Netatmo : New Module Camera, sid = ${sidModule} - ${module.type}`);
+            let newModuleCam;
+            if (module.type === 'NIS') {
+              // if siren module present, the device is created
+              newModuleCam = {
+                service_id: this.serviceId,
+                name: `Siren - ${moduleName}`,
+                selector: `netatmo:${sidModule}`,
+                external_id: `netatmo:${sidModule}`,
+                model: `netatmo-${module.type}`,
+                should_poll: false,
+                features: [
+                  {
+                    name: `Battery - ${moduleName}`,
+                    selector: `netatmo:${sidModule}:battery`,
+                    external_id: `netatmo:${sidModule}:battery`,
+                    category: DEVICE_FEATURE_CATEGORIES.BATTERY,
+                    type: DEVICE_FEATURE_TYPES.BATTERY.INTEGER,
+                    unit: DEVICE_FEATURE_UNITS.PERCENT,
+                    read_only: true,
+                    keep_history: true,
+                    has_feedback: true,
+                    min: 0,
+                    max: 100,
+                  },
+                  {
+                    name: `Détection intrusion - ${moduleName}`,
```

Comment:
so text is in french by default? This could be internationalized, or put in english first